src/utils/DateUtils.ts

```diff
@@ -29,111 +130,174 @@ class DefaultDateToStringOptions extends Options implements IDateToStringOptions
   predefinedFormat: string = undefined;
 }
 
+/**
+ * This class modifies the received Date so that it is rendered the way the options specify.
+ */
 export class DateUtils {
-  static convertFromJsonDateIfNeeded(date: string): Date;
-  static convertFromJsonDateIfNeeded(date: number): Date;
-  static convertFromJsonDateIfNeeded(date: Date): Date;
+
   static convertFromJsonDateIfNeeded(date: any): Date {
```

Comment:
I would add a standard comment ` // ` to explain why this function exists and calls `convertToStandardDate` directly (for legacy reasons) and why it's not documented publicly.